kodilla-testing/src/test/java/com/kodilla/testing/collection/CollectionTestSuite.java

```diff
@@ -0,0 +1,43 @@
+package com.kodilla.testing.collection;
+
+import org.junit.jupiter.api.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@DisplayName("Collection Test Suite")
+public class CollectionTestSuite {
+
+    @BeforeEach
+    public void before() {
+        System.out.println("Test Case: begin");
+    }
+
+    @AfterEach
+    public void after() {
+        System.out.println("Test Case: end");
+    }
+
+    @DisplayName("Metoda testOddNumbersExterminatorEmptyList powinna zwrócić pustą.")
+
+    @Test
+    void testOddNumbersExterminatorEmptyList() {
+        List<Integer> numbers = new ArrayList<>();
+        OddNumbersExterminator oddNumbersExterminator = new OddNumbersExterminator(0);
+        System.out.println("Testuje rozmiar listy: " + oddNumbersExterminator.exterminate(numbers));
+        Assertions.assertEquals(0, oddNumbersExterminator.exterminate(numbers).size());
+    }
+
+    @DisplayName("Metoda testOddNumbersExterminatorNormalList powinna zwrócić listę z liczbami parzystymi.")
+
+    @Test
+    void testOddNumbersExterminatorNormalList() {
+        List<Integer> numbers = new ArrayList<>();
+        for (int i = 0; i < 50; i++) {
```

Comment:
zadnych for/if/while/switch w metodach testowych. Jak juz chcemy miec jakies dane do testow, to piszemy jakas metode prywatna typu init(). 
W tym przypadku najlepiej miec konkretna liste, czyli np.: List<Integer> numbers = Arrays.asList(1, 2, 3, 4, 5);